docs/conf/eu/2017/news/2week-info.rst

```diff
@@ -0,0 +1,45 @@
+:template: 2017/eu-content.html
+
+.. post:: August 28, 2017
+   :tags: 2017, prague, practical information, tickets
+
+Get Ready for Write the Docs Prague 2017
+========================================
+
+We hope everyone is sufficiently excited for the conference, which is now just about two weeks away!
+
+Schedule and Activities Summary
+-------------------------------
+
+Our website is up-to-date with the latest details about sessions, activities, and locations. However, we are also happy to share with you a quick summary right here:
+
+- **Saturday September 9th - Boat Tour of Prague**. We will meet at `Prague Boats, pier no. 5 <https://goo.gl/maps/bqLP3VaytVo>`_ on **Saturday, September 9 at 13:45** to board the boat. We must leave the pier at **14:00** sharp (captain's rules) so unfortunately we cannot wait for late arrivals. All the details are in our `Boat Tour <http://www.writethedocs.org/conf/eu/2017/boat/>`_ page.
+
+- **Sunday September 10th - Writing Day and Welcome Reception**. Both the `Writing Day <http://www.writethedocs.org/conf/eu/2017/writing-day/>`_ and the Welcome Reception will be held at the main conference venue, `Auto Klub, Opletalova 1337/29, Praha 1 <https://goo.gl/maps/dcbYPV5x5DN2>`_. We will also hold a `Welcome Wagon <http://www.writethedocs.org/conf/eu/2017/welcome-wagon/>`_ orientation during the receiption to help ease your way into the conference.
```

Comment:
Maybe expand a bit more about the writing day? "Come with your laptop and work with others on documenting some open source projects, or your own"?

Reply:
We already wrote this in other places, I am trying to keep the post from getting long-winded, and we are already linking to the full writing day description...